docs/graphql.md

```diff
@@ -0,0 +1,46 @@
+# Using GraphQL
+
+\_Note: If testing this locally, you must have telescope running locally as well.
+
+# Adding/Modifying Resolvers for GraphQL
+
+1. Our resolvers are can be found in src/backend/web/graphql/index.js under module.exports.resolvers .
+
+## Step-by-step Guide
+
+1. To access GraphQL, append /grpahql to the end of your local app URI. For example if your Telescope app is running at http://localhost:3000, you can access GraphQL for telescope at http://localhost:3000/graphql
+
+2. A list of available endpoints for Telescope can be found in src/backend/web/graphql/index.js under module.exports.resolvers. The list of available resolvers may change over time.
+
+3. We will use the getPosts query, this query has two parameters page and perPage. For example, if I want to see the results of page 10 and have 30 results per page I would type the following:
+
+```
+{
+  getPosts(page: 10, perPage: 30) {
+
+  }
+}
+```
+
+This query won't run, because we also need to specify query fields. to see all available query fields type q within the getPosts() block:
+
+```
+{
+  getPosts(page: 10, perPage: 30) {
+    q
+  }
+}
+```
+
+An available list of query fields will be displayed. The following will let us retrieve the guid and author of all posts on the 10th page of results with 30(Posts) results shown:
+
+```
+{
+  getPosts(page: 10, perPage: 30) {
+    query
+    author
+  }
+}
```

Comment:
You can also mention that more than one query can be added to this request.
Like:

```
{
  getPosts(page: 10, perPage: 30) {
    query
    author
  }
  getFeeds {
     name
  }
}
```